cmake/modules/FindLibUSB.cmake

```diff
@@ -67,7 +67,7 @@ if(NOT LIBUSB_FOUND)
     if(WIN32 OR MSVC OR MINGW OR MSYS)
         find_package(7Zip REQUIRED)
 
-        set(LIBUSB_WIN_VERSION 1.0.22)
+        set(LIBUSB_WIN_VERSION 1.0.23)
```

Comment:
I think we should go to 1.0.20 here, unless there is no significant reason not to do so.

Reply:
My opinion is we should use the newest libusb that we can. So not see reasons to downgrade.

Reply:
Let me have a look at how this check is implemented at all...

Reply:
Yes, we should get the newest version here, if no previous version >= 1.0.20 is detected.